### PR TITLE
[JBIDE-19681] writing N&N for OpenShift 3.0.0.Alpha2

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Comment by adietish** [comment]
@xcoulon please dont merge it yet, @jcantrill will add further content

File: documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+= OpenShift What's New in OpenShift 3.0.0.Alpha2
+:page-layout: whatsnew
+:page-component_id: openshift
+:page-component_version: 3.0.0.Alpha2
+:page-product_id: jbt_core 
+:page-product_version: 4.3.0.Alpha2
+
+== Connect to OpenShift v3 servers
+Starting with JBoss Tools 4.3.0.Alpha2 we added support for OpenShift v3. When connecting you can now choose between a v2 or v3 server. + 
+image::./images/new-connection-wizard.png[] +
```
Comment (maxandersen) [inline documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc]: the +'s here are unnecessary afaics and makes the image not render.

File: documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+If you dont about the server type simply pick "Automatic" and we will figure for out. +
```
Comment (maxandersen) [inline documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc]: If you *do not know* about

File: documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+image::./images/connection-wizard-server-type.png[]
+
+related_jira::JBIDE-19096[]
+== Browse OpenShift v3 resources
+In JBoss Tools 4.3.0.Alpha2 you can now browse the resources of your OpenShift v3 server once you connected to it. 
+Decorations and child items unveil further informations about the given resource.+
+image::
```
Comment (maxandersen) [inline documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc]: image missing and bad + usage

File: documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+related_jira::JBIDE-19000[]
+
+== Display OpenShift v3 resource properties
+Whenever you select an OpenShift v3 resource in the OpenShift Explorer the properties view will show you further details for it.
+image::
```
Comment (maxandersen) [inline documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc]: image missing and bad + usage

File: documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+related_jira::JBIDE-19006[]
+
+== New OpenShift v3 java client library
+Jeff Cantrill developed a new restfull java client library to talk to the OpenShift v3 backend. This all brand-new client library is hosted at 
+
+  https://github.com/openshift/openshift-restclient-java
```
Comment (maxandersen) [inline documentation/whatsnew/openshift/openshift-news-3.0.0.Alpha2.adoc]: image missing and bad + usage

**Comment by adietish** [comment]
content by jeff is now included. Ready to be merged.

**Comment by maxandersen** [comment]
none of the images are referenced properly. nothing loads afaics.

**Comment by maxandersen** [comment]
seems the previous fixes with image urls was overwritten. I've redone those and pushed this to master.

**Pull request closed without merging**